mlrun/api/utils/clients/iguazio.py

```diff
@@ -397,14 +398,19 @@ def _create_project_in_iguazio(
     ) -> bool:
         _, job_id = self._post_project_to_iguazio(session, body)
         if wait_for_completion:
-            logger.debug(
+            self._logger.debug(
                 "Waiting for project creation job in Iguazio",
                 name=name,
                 job_id=job_id,
             )
             self._wait_for_job_completion(
                 session, job_id, "Project creation job failed"
             )
+            self._logger.debug(
+                "Waiting for project creation job in Iguazio",
+                name=name,
+                job_id=job_id,
+            )
```

Comment:
Why duplicate the log line?
In `_wait_for_job_completion` - either the job is completed or an exception is raised.

Reply:
to see a progress? it usually quick enough not to appear at all

Reply:
But what is the difference between this and the log in line 401?
It is the same exact thing and will not help with debugging.. at least say "Before / After" so we can differentiate the 2.
Although I still fail to see the point 🤷